TileBakeLibrary/BinaryMeshData.cs

```diff
@@ -80,10 +81,20 @@ public void ExportData(Tile tile)
                     {
                         mesh.vertices.Add(ConvertVertex(subobject.vertices[i]));
                         mesh.normals.Add(ConvertNormaltoBinary(subobject.normals[i]));
+                        
+                    }
+
+                    if (exportUVs)
+                    {
+                        //optional uvs
+                        for (int i = 0; i < subobject.uvs.Count; i++)
```

Comment:
gaat het hier mis als aangegeven wordt dat Uv's geexporteerd moeten worden maar er een subobject tussen zit zonder uv's?
wordt dit eerder ondervangen?  of moeten we er hier voor zorgen dat het subobject net zoveel uv's als vertices heeft (desnoods met een uv-coordinaat van 0,0)